### DISCUSSION TITLE
Why have separate fint and hgap parameters?

### DISCUSSION THREAD

**Discussion by DavidSagan** [MethodicalAcceleratorDesign/MAD-X#1230] [Q&A]
This is basically an historical question. In every paper I have seen, for a bend, the fint and hgap parameters are always combined in expressions in the form fint * hgap. This being the case, it has never made sense to me to have separate parameters. That is, why not just have a fintgap parameter equal to fint *hgap? Does anyone know what the reason was that MAD went with two parameters?

**Comment by rdemaria** [comment]
My understanding is that this allows to have a good default for `fint`  that works for any gap.
